src/services/api-gateway/apps/api-gateway/src/grpc-database-client.options.ts

```diff
@@ -0,0 +1,10 @@
+import { Transport, ClientOptions } from '@nestjs/microservices';
+import { join } from 'path';
+
+export const grpcDatabaseClientOptions: ClientOptions = {
+  transport: Transport.GRPC,
+  options: {
+    package: 'database',
+    protoPath: join(__dirname, '../../../node_modules/interfaces/database/database.proto'),
```

Comment:
Do you think there is a way avoiding loading them like this ? But from the lib like an 'import proto from "interfaces/database'.

Reply:
Yes i don't really like this either, but just checked and Nest only accept a protoPath as string in options :/